dom-manipulation/main.js

```diff
@@ -0,0 +1,22 @@
+var numberOfClick = 0;
+var buttonishot = document.querySelector('.hot-button');
+var clickcounter = document.querySelector('.click-count');
+function handleClick(event) {
+  numberOfClick++;
+  console.log(numberOfClick);
+  clickcounter.textContent;
+  if (numberOfClick.className < 4) {
+    numberOfClick.className = "cold"
+  } else if (numberOfClick.className < 7) {
+    numberOfClick.className = "cool"
```

Comment:
you need to update the classname of the hot-button element, as that is the one which needs to change color